hw02/EnemyCollision.cs

```diff
@@ -18,39 +18,74 @@ void Update()
     
     // TODO: Problem 7: Why did I not include the words public or private here?
     // What does it mean when I only write void (Movement)? What does void mean?
-    void Movement ()
+    void Movement () // "void" is used to specify a type of function that does NOT return a value
+    // public objects can be accessed from anywhere, but private objects can only be accessed from within the same class
+
     {
         // TODO: Problem 8: Please explain what the next 4 lines mean.
         // What is Input.GetAxis, transform.Translate, and transform.Rotate? 
-        float forwardMovement = Input.GetAxis("Vertical") * speed * Time.deltaTime;
-        float turnMovement = Input.GetAxis("Horizontal") * turnSpeed * Time.deltaTime;
+        float forwardMovement = Input.GetAxis("Vertical") * speed * Time.deltaTime; // this line defines a variable as the product of speed and vertical input position, all over time
```

Comment:
That's correct! The Input.GetAxis() will return -1 or 1, basically serving as a signed direction (going forward or backward)